### DISCUSSION TITLE
component import model as .gltf instead of spline url

### DISCUSSION THREAD

**Discussion by asharifauzan** [splinetool/react-spline#112] [Q&A]
Hi, I want to build webpage which display my 3d model and users are don't need internet access to see it. Is there an attribute which have similar behaviour with `src` of `<img />` HTML tag?

what currently I am using:
```
<div style={{width: "100vw", height: "100vh"}}>
  <Spline scene="https://prod.spline.design/xxxxx/scene.splinecode" />
</div>
```

what i want:
```
<div style={{width: "100vw", height: "100vh"}}>
  <Spline scene="./my3dmodel.gltf" />
</div>
```

thxu in advance

**Comment by marcofugaro** [comment]
You have to import the model into the Spline editor first, and then you can export it to react-spline.

Otherwise you can use model visualizing tools such as [model-viewer](https://modelviewer.dev/)
